css/style.css

```diff
@@ -0,0 +1,418 @@
+}
+
+.features_slogan {
+  font-size: 55px;
+  line-height: 55px;
+  font-weight: 500;
+  color: var(--basic-black);
+}
+
+.features_text {
+  color: var(--basic-dark);
+}
+
+/* buttons */
+
+.button {
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--basic-white);
+  background-color: var(--inactive-btn-color);
+  text-align: center;
+  text-decoration: none;
+  font-weight: 500;
+  text-transform: uppercase;
+  border: none;
+}
+
+.button:hover {
+  background-color: var(--hover-btn-color);
+}
+
+.button:active {
+  background-color: var(--active-btn-color);
+  color: var(--basic-active);
+}
+
+/* Services */
+
+.index_columns {
+  text-align: center;
+}
+
+.services_list {
+  list-style: none;
+}
+
+.services_item h3 {
+  font-size: 24px;
+  line-height: 30px;
+  font-weight: 700;
+  color: var(--basic-black);
+  text-transform: uppercase;
+}
+
+.services_item p {
+  color: var(--basic-dark);
+}
+
+.services_button:active {
+  color: var(--basic-active);
+}
+
+.apps_button {
+  background-color: var(--inactive-btn-apps-color);
+}
+
+.apps_button:hover {
+  background-color: var(--hover-btn-apps-color);
+}
+
+.apps_button:active {
+  background-color: var(--active-btn-apps-color);
+}
+
+.presentations_button {
+  background-color: var(--inactive-btn-presentation-color);
+}
+
+.presentations_button:hover {
+  background-color: var(--hover-btn-presentation-color);
+}
+
+.presentations_button:active {
+  background-color: var(--active-btn-presentation-color);
+}
+
+/* About */
+
+.about_slogan {
+  font-size: 45px;
+  line-height: 45px;
+  font-weight: 500;
+  color: var(--basic-black);
+}
+
+.about_orders {
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.development {
+  list-style: none;
+}
+
+.about_table {
+  font-weight: 700;
+}
+
+.about_table_caption {
+  color: var(--basic-black);
+}
+
+.about_table_percents {
+  font-size: 45px;
+  line-height: 64px;
+  color: var(--basic-black);
+}
+
+.about_table_percents td {
+  line-height: 18px;
+}
+
+/* Partners */
+
+.partners_list {
+  text-align: center;
+  list-style: none;
+}
+
+.partners_img {
+  color: var(--basic-black);
+  opacity: 0.2;
+}
+
+.partners_img:hover {
+  color: var(--basic-black);
+  opacity: 1;
+}
+
+.partners_img:active {
+  color: var(--basic-black);
+  opacity: 0.1;
+}
+
+/* Modal */
+
+.modal {
+  background-color: var(--basic-white);
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--basic-black);
+}
+
+.modal_header {
+  font-size: 45px;
+  line-height: 53px;
+}
+
+.modal_field {
+  border: 2px solid var(--inactive-input-border-color);
+}
+
+.modal_field:hover {
+  border-color: var(--hover-input-border-color);
+}
+
+.modal_form input:focus {
+  border-color: var(--basic-black);
+}
+
+.modal_form input::placeholder,
+.modal_form input::placeholder:hover,
+.modal_form input::placeholder:focus {
+  color: var(--special-gray);
+}
+
+.modal_form input::placeholder:invalid {
+  color: var(--error-input-border-color);
+}
```

Comment:
```suggestion
.modal_form input:invalid {
  color: var(--error-input-border-color);
}
```
Красным подсвечивается невалидный текст, а не плейсходлер
![image](https://user-images.githubusercontent.com/30810655/107642700-44c34f00-6c86-11eb-89df-246fad2311ed.png)